### PR TITLE
merging changes from NCO release of the code as 2.0.14

### PR DESCRIPTION
**Pull request by edwardhartnett**
Some programmers have change NCEPLIBS-prod_util code outside the repo.

This PR applies these changes. See #58 for the first attempt.

Part of #60

### CODE REVIEW

File: ush/err_exit [hunk 1]
```diff
@@ -1,13 +1,12 @@
-#!/bin/sh
+#!/bin/bash
 
-# SCRIPT NAME:  err_exit  Author:   Bill Facey   2 Apr 96
-#                         Modified: Kit Menlove 28 May 15
```
Comment (edwardhartnett) [inline ush/err_exit:4]: I don't think you should take out the names of the original authors of the code.

File: ush/getsystem [hunk 2]
```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
```
Comment (edwardhartnett) [inline ush/getsystem:1]: In NCEPLIBS we have a documentation standard: every file is documented, with a sentence or two of description, and the name of the programmers who worked on the code.

Do we know who made the changes to this file, so we can get that information in?

**Comment by edwardhartnett** [comment]
@natalie-perlin for some reason GitHub won't let me request you to review this PR. It should contain all the changes you want from #58 . Please take a look at it...

File: ush/getsystem (hunk 2 above)
Comment (natalie-perlin) [reply ush/getsystem:1]: As the new file (change from *.pl)  comes not from the GitHub but from the NCO website, there is no additional info on particular developers who modified/recodeded the script from perl to python.

File: ush/err_exit (hunk 1 above)
Comment (natalie-perlin) [reply ush/err_exit:4]: Sounds fair to leave the original authors, this was the change coming from https://www.nco.ncep.noaa.gov/pmb/codes/nwprod/prod_util.v2.0.14/

**Comment by natalie-perlin** [comment]
@edwardhartnett - all looks good! 
I was able to test this #59 on Orion: build, install, and successfully run tests `test_mdate.sh `and `test_ndate.sh`. 
The third test, `test_nhour.sh`, requires an additional utility **faketime** that is not at the moment.

File: ush/err_exit (hunk 1 above)
Comment (edwardhartnett) [reply ush/err_exit:4]: OK, we'll just leave it exactly as they had it, to make it easier to merge future changes.

File: ush/getsystem (hunk 2 above)
Comment (AlexanderRichert-NOAA) [reply ush/getsystem:1]: As it it happens, that was me that created the python version of gestsystem in/around March 2021.

Comment (edwardhartnett) [reply ush/getsystem:1]: @AlexanderRichert-NOAA is there anything you can't do? ;-)

How about a one-sentence description for the documentation?

**Comment by edwardhartnett** [comment]
@AlexanderRichert-NOAA if you can approve this PR I'll merge this and do a release...

**Review by AlexanderRichert-NOAA — APPROVED** [review]

**Comment by edwardhartnett** [comment]
@natalie-perlin since this library was somehow released by NCO with a different version number, I'll change the number of the next NCEPLIBS-prod_util release to 2.1.0. Does that make sense?

Are there any other changes you would like to get in before the release?

**Comment by natalie-perlin** [comment]
@edwardhartnett - sounds good to me. 
So it sounds that the NCO version and EMC version would diverge. 
Could this equivalency between NCO v.2.0.14 and EMC v2.1.0 be documented somehow, just to avoid the confusion. See, e.g. request from @chan-hoo https://github.com/ufs-community/ufs-srweather-app/issues/839

@chan-hoo , FYI

**Comment by edwardhartnett** [comment]
The inclusion of code from the NCO 2.0.14 release is documented in the release notes.

However, there should be no one else releasing this package. There should only be releases from this repo.

**Comment by chan-hoo** [comment]
> @edwardhartnett - sounds good to me. So it sounds that the NCO version and EMC version would diverge. Could this equivalency between NCO v.2.0.14 and EMC v2.1.0 be documented somehow, just to avoid the confusion. See, e.g. request from @chan-hoo [ufs-community/ufs-srweather-app#839](https://github.com/ufs-community/ufs-srweather-app/issues/839)
> 
> @chan-hoo , FYI

@natalie-perlin, thanks for the info. Could you let me how I can load the recent version of `prod_util` (>v.2) on Hera, Orion, or Jet?

**Pull request merged**